Cargo.toml

```
@@ -50,7 +50,7 @@ socket2 = {version = "0.4.7", features = ["all"]}
 spin = "0.9.4"
 tokio = {version = "1.24", features = ["net", "io-util", "macros", "sync"]}
 tokio-boring = "2.1.5"
-tokio-tungstenite = {version = "0.17", features = ["stream"], default-features = false}
+tokio-tungstenite = {version = "0.18", features = ["stream"], default-features = false}
```

This should fix the build.

```suggestion
tokio-tungstenite = {version = "0.18", features = ["handshake"], default-features = false}
```

Refer to: https://github.com/snapview/tokio-tungstenite/compare/v0.17.2...v0.18.0

---

Or as in [tokio-tungstenite's README](https://github.com/snapview/tokio-tungstenite#usage):

```suggestion
tokio-tungstenite = "*"
```